woocommerce/woocommerce-ios
# Downloadable Products: Fix issue uploading device documents

Closes: #11446 


## Description

This PR fixes the issue with uploading documents from physical devices. This was fixed by adding `startAccessingSecurityScopedResource` to get access to local files outside of the app's container.

## Testing instructions

- Build the app to a physical device.
- Log in to a store and select the Products tab.
- Select an existing product or create a new one.
- Select Add more details > Downloadable products.
- Select Add file > Document on device.
- Select any document and notice that the upload succeeds.

## Screenshots

Not included due to privacy reasons (not to show files in my phone).

---
